src/main/webapp/src/components/entityAnalysisInfo.js

```diff
@@ -81,23 +81,55 @@ class EntityAnalysisInfo extends React.Component {
       return <p>No entities were found.</p>;
     }
 
-    /** Gather the most important words and their data in simple arrays. */
+    /** Generate a custom tooltip for each entity */
+    function getTooltip(entity) {
+      return '<div style="font-size:15px">' + entity.name + '<br>' 
+          + '<b>Type: </b>' + entity.type + '<br>' 
+          + '<b>Salience: </b>' + entity.salience + '<br>' 
+          + ((entity.wikiLink !== "") ? 'Click for wiki link' : '')
+          + '</div>';
+    }
+
     var top10WordsData = [];
     var top10WordsLinks = [];
-    top10WordsData.push(['Word and Type', 'Importance'])
+    /** 
+     * Create a custom tooltip to overwrite the default one, we want to show whether a wiki link
+     * is available, besides the default data.
+     * This tooltip will be able to interpret HTML, so we can format the text.
+     */
+    const customTooltip = { role: "tooltip", type: "string", 'p': {'html': true} };
```

Comment:
what is `p` and why do we need the single quotes only for it?

Reply:
I must say that's just how I found it in multiple sources. Apparently you can add single quotes to the entire object. The p element tells the tooltip to interpret the tooltip text as HTML instead of the default interpretation.